docs/Other SDL Features/Using App Services/index.md

```diff
@@ -284,6 +314,28 @@ sdlManager.sendRPC(unsubscribeServiceData);
 ```
 !@
 
+@![javascript]
+```js
+// Get service data once
+const getAppServiceData = new SDL.rpc.messages.GetAppServiceData()
+    .setServiceType(SDL.rpc.enums.AppServiceType.MEDIA);
+
+// Subscribe to future updates if you want them
+getAppServiceData.setSubscribe(true);
+
+const response = await sdlManager.sendRpc(getAppServiceData).catch(error => error);
+if (response !== null) {
```

Comment:
`response` could be a `GenericResponse` on head units that don't understand the RPC (which could be a lot since it's a newer feature), then your `gets` will fail I think

Reply:
sendRpc checks for matching function ids for the request and response, but even though the response wouldn't be a GenericResponse, there could be another issue as a result of that. I'll add an issue to the JS Suite repo for this case.